generator_zip.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: UTF-8 -*-
```

Comment:
Zastawiam, się, czy nie połączyć tych dwóch skryptów.
Coś tam już dziergałem w tym temacie.
Dasz mi 8 godzin?

Reply:
Jasne, możesz kombinować. To raczej ciężkie nie będzie więc mógłbym to nawet ja zrobić

Reply:
Czemu nie, nie nadążam z pomysłami. Taką sraczkę intelektualną mam teraz :-P

Chciałem mieć (u siebie, w mrknow zacząłem) skrypt, który nie tylko policzy md5, zbuduje zipy, ale i pobierze zewnętrzne biblioteki i całość wypchnie na serwer.
Początki mam. Co prawda nie ma tutaj potrzeby wypychania, bo repo binarne jest. Ale reszta ma sens, mi się wydaje. 
Choć może zwyczajnie mnie ponosi...